### ISSUE TITLE
OSX: High Sierra: "Illegal Instruction: 4"

### ISSUE THREAD

**Issue by esaruoho** [schismtracker/schismtracker#131]
What could
```
Illegal Instruction: 4
```

mean? That's what I get when I run schismtracker in the terminal on a macOS High Sierra. I've tried brew installing a schismtracker but even though I did it, and it succeeded in installing, I still get the same error.

**Comment by jangler** [comment]
Looks like something to do with compiler optimizations that are non-portable across macOS versions. If you've built Schism from source and you're still getting the error, then reducing the compiler optimization level might fix it.

You can try this by going through the normal build process, but instead of running `../configure`, run `../configure CFLAGS='-O0 -g'`.

**Comment by esaruoho** [comment]
@jangler hi, i tried to download from https://github.com/schismtracker/schismtracker/releases and start the app, and got the same `Illegal Instruction: 4`

I also tried to git clone the github repo, and was completely unable to find what to do.

autoreconf doesn't exist on brew and on macosx.. autoconf does nothing.

```
Thu Jan 04 22:29:09 ~/work/schismtracker >> ./configure
./configure: line 2180: syntax error near unexpected token `-Wall'
./configure: line 2180: `AM_INIT_AUTOMAKE(-Wall subdir-objects)'
```

so i'm really not sure how to proceed. i tried `./configure CFLAGS='-O0 -g'` and it gave the same errors.

so i'm not able to get from homebrew, and the "newest build" doesn't work either.. what should i try next?

the instructions on https://github.com/schismtracker/schismtracker/blob/master/docs/building_on_osx.md consist of absolutely 0 command line commands.would be swell to see actual documentation?

**Comment by jangler** [comment]
The command line sequence should be the same as in https://github.com/schismtracker/schismtracker/blob/master/docs/building_on_linux.md (maybe that should be made clearer). I'm not sure what to do if you don't have autoreconf, though. You can absolutely still compile it, but I don't know the autotools dance.

**Comment by esaruoho** [comment]
today i tried
`$ automake`
and got 
```
configure.ac: error: no proper invocation of AM_INIT_AUTOMAKE was found.
configure.ac: You should verify that configure.ac invokes AM_INIT_AUTOMAKE,
configure.ac: that aclocal.m4 is present in the top-level directory,
configure.ac: and that aclocal.m4 was recently regenerated (using aclocal)
Makefile.am:154: error: USE_X11 does not appear in AM_CONDITIONAL
Makefile.am:157: error: USE_XV does not appear in AM_CONDITIONAL
Makefile.am:163: error: USE_ALSA does not appear in AM_CONDITIONAL
Makefile.am:165: error: USE_ALSA_DLTRICK does not appear in AM_CONDITIONAL
Makefile.am:173: error: USE_OSS does not appear in AM_CONDITIONAL
Makefile.am:178: error: USE_MMAP does not appear in AM_CONDITIONAL
Makefile.am:182: error: USE_WIN32 does not appear in AM_CONDITIONAL
Makefile.am:194: error: HAVE_WINDRES does not appear in AM_CONDITIONAL
Makefile.am:197: error: HAVE_GIT does not appear in AM_CONDITIONAL
Makefile.am:211: error: USE_WII does not appear in AM_CONDITIONAL
Makefile.am:216: error: USE_MACOSX does not appear in AM_CONDITIONAL
Makefile.am:225: error: USE_NETWORK does not appear in AM_CONDITIONAL
Makefile.am:232: error: NEED_ASPRINTF does not appear in AM_CONDITIONAL
Makefile.am:235: error: NEED_VASPRINTF does not appear in AM_CONDITIONAL
Makefile.am:238: error: NEED_MEMCMP does not appear in AM_CONDITIONAL
Makefile.am:241: error: NEED_STRPTIME does not appear in AM_CONDITIONAL
Makefile.am:244: error: NEED_MKSTEMP does not appear in AM_CONDITIONAL
configure.ac:31: error: required file 'config.h.in' not found
Makefile.am:250: warning: source file 'schism/page_blank.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
automake: warning: possible forward-incompatibility.
automake: At least a source file is in a subdirectory, but the 'subdir-objects'
automake: automake option hasn't been enabled.  For now, the corresponding output
automake: object file(s) will be placed in the top-level directory.  However,
automake: this behaviour will change in future Automake versions: they will
automake: unconditionally cause object files to be placed in the same subdirectory
automake: of the corresponding sources.
automake: You are advised to start using 'subdir-objects' option throughout your
automake: project, to avoid future incompatibilities.
Makefile.am:250: warning: source file 'schism/charset.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/sample-edit.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/dmoz.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_info.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_palette.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_instruments.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_log.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_about.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/pattern-view.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/xpmdata.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/menu.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/volume-core.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/disko.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_loadinst.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/mplink.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/clippy.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_loadmodule.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_vars.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/palettes.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/compression.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/generic.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mmcmp.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/669.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/aiff.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/ams.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/au.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/f2r.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/far.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/imf.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/it.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/iti.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/its.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/liq.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mdl.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/med.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mf.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mid.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mod.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mt2.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mtm.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/mus.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/ntk.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/okt.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/pat.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/raw.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/s3i.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/s3m.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/sfx.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/stm.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/ult.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/wav.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/xi.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'fmt/xm.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/util.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_midi.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/draw-char.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_help.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/slurp.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/widget-keyhandler.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/main.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_midiout.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_message.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_loadsample.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/dialog.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_preferences.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/widget.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/config.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/status.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/video.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/sample-view.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_patedit.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_config.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/keyboard.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_samples.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/itf.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_orderpan.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/page_waterfall.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/midi-core.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/midi-ip.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/audio_playback.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/config-parser.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/audio_loadsave.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/draw-misc.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/fakemem.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'schism/version.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/csndfile.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/mixutil.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/equalizer.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/mixer.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/filters.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/fmopl2.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/fmopl3.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/fmpatches.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/sndmix.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/opl-util.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/snd_fm.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/effects.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/snd_gm.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:250: warning: source file 'player/tables.c' is in a subdirectory,
Makefile.am:250: but option 'subdir-objects' is disabled
Makefile.am:217: warning: source file 'sys/macosx/macosx-sdlmain.m' is in a subdirectory,
Makefile.am:217: but option 'subdir-objects' is disabled
Makefile.am:217: warning: source file 'sys/macosx/ibook-support.c' is in a subdirectory,
Makefile.am:217: but option 'subdir-objects' is disabled
Makefile.am:217: warning: source file 'sys/macosx/midi-macosx.c' is in a subdirectory,
Makefile.am:217: but option 'subdir-objects' is disabled
Makefile.am:217: warning: source file 'sys/macosx/volume-macosx.c' is in a subdirectory,
Makefile.am:217: but option 'subdir-objects' is disabled
Makefile.am:217: warning: source file 'sys/macosx/osdefs.c' is in a subdirectory,
Makefile.am:217: but option 'subdir-objects' is disabled
Makefile.am:164: warning: source file 'sys/alsa/init.c' is in a subdirectory,
Makefile.am:164: but option 'subdir-objects' is disabled
Makefile.am:164: warning: source file 'sys/alsa/volume-alsa.c' is in a subdirectory,
Makefile.am:164: but option 'subdir-objects' is disabled
Makefile.am:164: warning: source file 'sys/alsa/midi-alsa.c' is in a subdirectory,
Makefile.am:164: but option 'subdir-objects' is disabled
Makefile.am:174: warning: source file 'sys/oss/volume-oss.c' is in a subdirectory,
Makefile.am:174: but option 'subdir-objects' is disabled
Makefile.am:174: warning: source file 'sys/oss/midi-oss.c' is in a subdirectory,
Makefile.am:174: but option 'subdir-objects' is disabled
Makefile.am:183: warning: source file 'sys/win32/osdefs.c' is in a subdirectory,
Makefile.am:183: but option 'subdir-objects' is disabled
Makefile.am: error: object 'osdefs.$(OBJEXT)' created by 'sys/win32/osdefs.c' and 'sys/macosx/osdefs.c'
Makefile.am:183: warning: source file 'sys/win32/slurp-win32.c' is in a subdirectory,
Makefile.am:183: but option 'subdir-objects' is disabled
Makefile.am:183: warning: source file 'sys/win32/volume-win32mm.c' is in a subdirectory,
Makefile.am:183: but option 'subdir-objects' is disabled
Makefile.am:183: warning: source file 'sys/win32/midi-win32mm.c' is in a subdirectory,
Makefile.am:183: but option 'subdir-objects' is disabled
Makefile.am:183: warning: source file 'sys/win32/filetype.c' is in a subdirectory,
Makefile.am:183: but option 'subdir-objects' is disabled
Makefile.am:183: warning: source file 'sys/win32/localtime_r.c' is in a subdirectory,
Makefile.am:183: but option 'subdir-objects' is disabled
Makefile.am:155: warning: source file 'sys/x11/xscreensaver.c' is in a subdirectory,
Makefile.am:155: but option 'subdir-objects' is disabled
Makefile.am:155: warning: source file 'sys/x11/xkb.c' is in a subdirectory,
Makefile.am:155: but option 'subdir-objects' is disabled
Makefile.am:158: warning: source file 'sys/x11/xv.c' is in a subdirectory,
Makefile.am:158: but option 'subdir-objects' is disabled
Makefile.am:233: warning: source file 'sys/stdlib/asprintf.c' is in a subdirectory,
Makefile.am:233: but option 'subdir-objects' is disabled
Makefile.am:236: warning: source file 'sys/stdlib/vasprintf.c' is in a subdirectory,
Makefile.am:236: but option 'subdir-objects' is disabled
Makefile.am:239: warning: source file 'sys/stdlib/memcmp.c' is in a subdirectory,
Makefile.am:239: but option 'subdir-objects' is disabled
Makefile.am:242: warning: source file 'sys/stdlib/strptime.c' is in a subdirectory,
Makefile.am:242: but option 'subdir-objects' is disabled
Makefile.am:245: warning: source file 'sys/stdlib/mkstemp.c' is in a subdirectory,
Makefile.am:245: but option 'subdir-objects' is disabled
Makefile.am:179: warning: source file 'sys/posix/slurp-mmap.c' is in a subdirectory,
Makefile.am:179: but option 'subdir-objects' is disabled
Makefile.am:212: warning: source file 'sys/wii/isfs.c' is in a subdirectory,
Makefile.am:212: but option 'subdir-objects' is disabled
Makefile.am:212: warning: source file 'sys/wii/osdefs.c' is in a subdirectory,
Makefile.am:212: but option 'subdir-objects' is disabled
Makefile.am: error: object 'osdefs.$(OBJEXT)' created by 'sys/wii/osdefs.c' and 'sys/macosx/osdefs.c'
Makefile.am:142: warning: source file 'auto/default-font.c' is in a subdirectory,
Makefile.am:142: but option 'subdir-objects' is disabled
Makefile.am:142: warning: source file 'auto/helptext.c' is in a subdirectory,
Makefile.am:142: but option 'subdir-objects' is disabled
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am: error: am__fastdepCC does not appear in AM_CONDITIONAL
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:   The usual way to define 'am__fastdepCC' is to add 'AC_PROG_CC'
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am: error: AMDEP does not appear in AM_CONDITIONAL
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:   The usual way to define 'AMDEP' is to add one of the compiler tests
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:     AC_PROG_CC, AC_PROG_CXX, AC_PROG_OBJC, AC_PROG_OBJCXX,
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:     AM_PROG_AS, AM_PROG_GCJ, AM_PROG_UPC
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am: error: am__fastdepOBJC does not appear in AM_CONDITIONAL
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:   The usual way to define 'am__fastdepOBJC' is to add 'AC_PROG_OBJC'
/usr/local/Cellar/automake/1.15.1/share/automake-1.15/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
```

so i'm not sure i'm making any progress.

@jangler regards "maybe that could be made clearer", is it Okay if I, _if I'm able to_ create a step-by-step on how to install schismtracker on OSX, make documentation _for_ OSX which _does not reference_ building on Linux?

I've now tried to follow the Linux instructions like this:

```
Fri Jan 05 10:14:02 ~/work/schismtracker >> mkdir -p build
Fri Jan 05 10:17:44 ~/work/schismtracker >> cd build
Fri Jan 05 10:17:46 ~/work/schismtracker/build >> ../configure && make
../configure: line 2180: syntax error near unexpected token `-Wall'
../configure: line 2180: `AM_INIT_AUTOMAKE(-Wall subdir-objects)'
```

and then your additions:

```
Fri Jan 05 10:17:50 ~/work/schismtracker/build >> ../configure CFLAGS='-O0 -g' && make
../configure: line 2180: syntax error near unexpected token `-Wall'
../configure: line 2180: `AM_INIT_AUTOMAKE(-Wall subdir-objects)'
Fri Jan 05 10:18:17 ~/work/schismtracker/build >> 
```

I don't know what a -Wall is, and I'm not sure how to proceed.

**Comment by esaruoho** [comment]
Made a PR #132  which fixes #131
